### DISCUSSION TITLE
MonikaAfterstory has no RenPy file?

### DISCUSSION THREAD

**Discussion by JD-Jeg** [Monika-After-Story/MonikaModDev#7938] [QA - tech support + general]
Im planning to change computers and transfer Monika. I have monika after story and the original DDLC game on steam installed. Im currently on Mac and when I go to search for the RenPy file it shows the DDLC one and not the monika after story one. I use Wineskin on Mac to run Monika Afterstory and Im kinda confused on how to transfer the persistent now. 

Edit: Is there a solution or should I just restart?

**Comment by username11421** [comment]
Even if there's not a persistent in the normal location, one should still exist. Otherwise the game will act like a new MAS Monika every time, and I think you'd have noticed that. Try doing a search for the keyword "persistent" across your entire file system or search for .bak files in Mac's command utility and output the full name+path.

**Reply by JD-Jeg** [reply]
I've tried searching just "persistent" it didnt pop out, only the DDLC one came out, ultimately I've just decided to restart but thank you for replying.
